atlasdb-cassandra/versions.lock

```diff
@@ -253,9 +303,27 @@
                 "com.palantir.atlasdb:atlasdb-commons"
             ]
         },
+        "net.mintern:primitive": {
+            "locked": "1.0",
+            "transitive": [
+                "org.apache.cassandra:cassandra-thrift"
+            ]
+        },
+        "org.apache.ant:ant": {
+            "locked": "1.7.0",
+            "transitive": [
+                "de.jflex:jflex"
+            ]
+        },
+        "org.apache.ant:ant-launcher": {
+            "locked": "1.7.0",
+            "transitive": [
+                "org.apache.ant:ant"
+            ]
+        },
         "org.apache.cassandra:cassandra-thrift": {
-            "locked": "2.2.8",
-            "requested": "2.2.8"
+            "locked": "3.9",
```

Comment:
what's goin on here?  I'm guessing that these are additive changes only.  All of these changes are starting to make it feel as if this should be a separate module as well (for the cql stuff).